src/ui/environment-entry.ui

```diff
@@ -2,101 +2,9 @@
-        <property name="expand">False</property>
-        <property name="fill">True</property>
-        <property name="position">0</property>
-      </packing>
-    </child>
-    <child>
-      <object class="GtkBox">
-        <property name="visible">True</property>
-        <property name="can-focus">False</property>
-        <property name="hexpand">True</property>
-        <property name="orientation">vertical</property>
-        <child>
-          <object class="GtkLabel" id="label_title">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="halign">start</property>
-            <property name="valign">end</property>
-            <property name="vexpand">True</property>
-            <property name="label">Custom</property>
-            <attributes>
-              <attribute name="font-desc" value="Cantarell 12"/>
-              <attribute name="weight" value="bold"/>
-            </attributes>
-          </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">0</property>
-          </packing>
-        </child>
-        <child>
-          <object class="GtkLabel" id="label_subtitle">
-            <property name="visible">True</property>
-            <property name="can-focus">False</property>
-            <property name="halign">start</property>
-            <property name="valign">start</property>
-            <property name="vexpand">True</property>
-            <property name="label">Environment description</property>
-            <attributes>
-              <attribute name="font-desc" value="Cantarell 10"/>
-            </attributes>
-            <style>
-              <class name="plain-label"/>
-            </style>
-          </object>
-          <packing>
-            <property name="expand">False</property>
-            <property name="fill">True</property>
-            <property name="position">1</property>
-          </packing>
-        </child>
-      </object>
-      <packing>
-        <property name="expand">False</property>
-        <property name="fill">True</property>
-        <property name="position">1</property>
-      </packing>
-    </child>
-    <child>
-      <object class="GtkImage" id="img_selected">
-        <property name="can-focus">False</property>
-        <property name="halign">center</property>
-        <property name="valign">center</property>
-        <property name="margin-start">5</property>
-        <property name="margin-end">5</property>
-        <property name="icon-name">object-select-symbolic</property>
-        <style>
-          <class name="selection-icon"/>
-        </style>
-      </object>
-      <packing>
-        <property name="expand">False</property>
-        <property name="fill">True</property>
-        <property name="position">2</property>
-      </packing>
-    </child>
```

Comment:
@mirkobrombin this PR is quite a trim, are you ok with it?

Also, this is my first time doing Python GTK, I don't think we need this `.ui` anymore, how can I make `BottlesEnvironmentEntry` extend Handy.ActionRow without this template?

Reply:
> @mirkobrombin this PR is quite a trim, are you ok with it?
> 
> Also, this is my first time doing Python GTK, I don't think we need this `.ui` anymore, how can I make `BottlesEnvironmentEntry` extend `Handy.ActionRow` without a template?
> 
> EDIT: nevermind, easy peasy

It's fine. This is also my first project using .ui with gtk